torchvision/models/feature_extraction.py

```diff
@@ -184,6 +184,23 @@ def _get_leaf_modules_for_ops() -> List[type]:
     return result
 
 
+def set_default_tracer_kwargs(original_tr_kwargs: Optional[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
Is this on purpose public? If it is it should have a documentation. If not then we should prepend an `_` to its name.

Reply:
it should be private, thanks for pointing out